src/basic/mount-util.c

```diff
@@ -689,3 +689,35 @@ int umount_verbose(const char *what) {
                 return log_error_errno(errno, "Failed to unmount %s: %m", what);
         return 0;
 }
+
+const char *mount_propagation_flags_to_string(unsigned long flags) {
+
+        switch (flags & (MS_SHARED|MS_SLAVE|MS_PRIVATE)) {
```

Comment:
All values which are assigned to `flags` are checked to be on of the four possibilities. It seems an assert would make sense:
```
case 0:
   return "";
default:
   assert_not_reached(...);
```